pyanaconda/packaging/yumpayload.py

```diff
@@ -1214,7 +1231,7 @@ def _applyYumSelections(self):
 
         if env:
             try:
-                self.selectEnvironment(env)
+                self._selectYumEnvironment(env)
```

Comment:
Probably not. I'll change that to match dnfpayload: leave the selectEnvironment call alone, and override selectEnvironment to call the parent method and then _selectYumEnvironment.